src/nvim/ex_docmd.c

```diff
@@ -603,6 +603,12 @@ int do_cmdline(char_u *cmdline, LineGetter fgetline,
                               cmd_getline, cmd_cookie);
     recursive--;
 
+    // If in preview mode then only the first command separated
+    // '|' should be executed.
```

Comment:
Brevity/concision:

    // Only do the first '|'-separated command in preview-mode ('inccommand').